src/WebDAVAdapter.php

```diff
@@ -275,7 +275,14 @@ public function getMimetype($path)
      */
     protected function normalizeObject(array $object, $path)
     {
-        if (! isset($object['{DAV:}getcontentlength']) or $object['{DAV:}getcontentlength'] == "") {
+        $hasNotContentLength = !isset($object['{DAV:}getcontentlength']) ||
+            $object['{DAV:}getcontentlength'] == "";
```

Comment:
On multiline conditions the operator should be on the start of the line, not at the end -> Better readability and you see immediately how it is supposed to be.

This is not javascript, where such silly syntax is needed :trollface: